fe/fe-core/src/main/java/com/starrocks/statistic/AnalyzeMgr.java

```diff
@@ -288,6 +288,7 @@ public void clearStatisticFromDroppedTable() {
 
         ConnectContext statsConnectCtx = StatisticUtils.buildConnectContext();
         statsConnectCtx.setThreadLocalInfo();
+        statsConnectCtx.setStatisticsConnection(true);
```

Comment:
can we move this step into `StatisticUtils.buildConnectContext()` ?

Reply:
may can't, query cache will use `buildConnectContext` too